src/patternfly/components/Sidebar/examples/Sidebar.md

```diff
@@ -29,7 +29,7 @@ import './Sidebar.css'
   {{#> sidebar-panel}}
     Sidebar panel. Adding some extra content so that the gap between the panel and content area is better illustrated in this example.
   {{/sidebar-panel}}
-  {{#> sidebar-content}}
+  {{#> sidebar-content sidebar-content--modifier="pf-m-bordersss"}}
```

Comment:
Do you mean `pf-m-border` on line 16 and 32?

Reply:
Oof, nope! Good catch, thanks

Reply:
@srambach updated!